src/uu/seq/src/seq.rs

```diff
@@ -212,7 +212,7 @@ fn write_value_float(
 ) -> std::io::Result<()> {
     let value_as_str = if *value == ExtendedBigDecimal::MinusZero && is_first_iteration {
```

Comment:
Does this first if statement still need to exist? I left it in because I am unsure about what is_first_iteration does